vortex-scalar/src/bigint/mod.rs

```diff
@@ -15,6 +16,7 @@ use num_traits::{CheckedAdd, CheckedSub, ConstZero, One, Zero};
 #[derive(Debug, Copy, Clone, Default, Eq, PartialEq, Hash, PartialOrd, Ord)]
 pub struct i256(arrow_buffer::i256);
 
+#[allow(clippy::same_name_method)]
```

Comment:
This is only the case if you compile with `arbitrary` feature which is only enabled in the fuzzer